src/azul/openapi/params.py

```diff
@@ -15,7 +15,16 @@
 )
 
 
-def path(name: str, type_: TYPE, **kwargs: PrimitiveJSON) -> JSON:
+class JsonParam:
```

Comment:
I'm dubious whether this is needed. Can't this just make use of the `JSON` alternative of the [`TYPE` union](https://github.com/DataBiosphere/azul/pull/1647/files#diff-80fb4e5dfa369b5d82f4d6a888f0d83bR22)?

Reply:
If the answer is complicated, don't respond here but rather jump on a call with me. I may also just want to play with this myself once the remaining issues are addressed.

Reply:
It's a little more complicated than that, but I was able to come up with a better, simpler solution anyway. The complication stems from the fact that OpenAPI uses the `content` property to describe JSON-formatted parameters instead of the `schema` property, so this case needs to handled separately from all non-JSON parameters.

Reply:
It appears that you were able to avoid adding another alternative to the type union exactly as I suspected. That's progress but I don't think you've found the most elegant solution yet, though. I'll comment on the latest changes.